android/capacitor/src/main/java/com/getcapacitor/JSInjector.java

```diff
@@ -69,12 +69,12 @@ public String getScriptString() {
      * @return
      */
     public InputStream getInjectedStream(InputStream responseStream) {
-        String js = "<script type=\"text/javascript\">" + getScriptString() + "</script>";
+        String js = "<script type=\"text/javascript\">" + getScriptString().replace("${", "\\${") + "</script>";
         String html = this.readAssetStream(responseStream);
         if (html.contains("<head>")) {
-            html = html.replace("<head>", "<head>\n" + js + "\n");
+            html = html.replaceFirst("<head>", "<head>\n" + js + "\n");
         } else if (html.contains("</head>")) {
-            html = html.replace("</head>", js + "\n" + "</head>");
+            html = html.replaceFirst("</head>", js + "\n" + "</head>");
```

Comment:
```
<head>
  <head>
  
  </head>

//consider some random stuff here.

</head>
```

I am not into the context much, but assuming the above code can be a possible value of `html` variable, do you think instead of `replaceFirst` we should replace last `</head>` tag?